extensions/wikia/ARecoveryEngine/js/PageFair/pageFair.js

```diff
@@ -33,10 +33,10 @@ define('ext.wikia.aRecoveryEngine.recovery.pageFair', [
 		return isBlocking;
 	}
 
-	function addMarker(slotId) {
-		var slot = document.getElementById(slotId);
-
-		slot.setAttribute('adonis-marker', '');
+	function addMarker(slotElement) {
+		if (slotElement) {
```

Comment:
yes, we should, but my thinking was not to throw an error in case we'll call addMarker and `doc.getElementById()` which will return null. But this may be overprotective and totally unneeded here. No strong feelings about that from my side.